backend/src/providers/payments/razorpay.provider.ts

```
@@ -0,0 +1,354 @@
+   */
+  async retrieveAccount(): Promise<RazorpayAccountInfo> {
+    // Razorpay key ID encodes the environment implicitly (rzp_test_ / rzp_live_)
+    return {
+      id: this.keyId,
+      merchantName: null, // requires dashboard API not available in OSS SDK
+      livemode: this.environment === 'live',
+    };
+  }
+
+  async listPlans(): Promise<RazorpayPlan[]> {
+    const response = await this.client.plans.all({ count: 100 }) as { items: RazorpayPlan[] };
```

ESLint

[error] Replace `await·this.client.plans.all({·count:·100·}` with `(await·this.client.plans.all({·count:·100·})`

(prettier/prettier)